snap/snapcraft.yaml

```diff
@@ -65,8 +72,22 @@ parts:
       - libegl1
       - libegl1-mesa
       - libglx-mesa0
+    build-attributes:
+      - enable-patchelf
+    stage:
+      # The libraries in .../dri need no-patchelf, so they come from the mesa-unpatched part
+      - -usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/dri
+
+  mesa-gl1-dri:
+    plugin: nil
+    stage-packages:
+      - libgl1-mesa-dri
```

Comment:
Should this package be removed from the `mesa` part above?

Reply:
it should bring in the dependencies for it though, which is part of the trick, also I took this from https://github.com/MirServer/egmde-snap/blob/main/snap/snapcraft.yaml